convert2rhel/pkghandler.py

```diff
@@ -72,6 +72,7 @@ def call_yum_cmd_w_downgrades(cmd, pkgs, retries=MAX_YUM_CMD_CALLS):
     dependencies, especially when it tries to downgrade pkgs. This function
     tries to resolve the dependency errors where yum is not able to.
     """
+    # TODO(r0x0d): Verify if we need to remove this specific function.
```

Comment:
Link Jira issue once created
```suggestion
    # TODO(r0x0d): Verify if we need to remove this specific function. https://issues.redhat.com/browe/RHELC-
```

Reply:
noted.